packages/amplication-client/src/Project/ArchitectureConsole/ModelOrganizerToolbar.tsx

```diff
@@ -63,16 +64,18 @@ export default function ModelOrganizerToolbar({
   );
   const [confirmChanges, setConfirmChanges] = useState<boolean>(false);
   const [changesDialog, setChangesDialog] = useState<boolean>(false);
-  const [createError, setCreateError] = useState<boolean>(false);
+  const [applyChangesErrorMessage, setApplyChangesErrorMessage] = useState<
+    string | null
+  >(null);
 
   const handleConfirmChangesState = useCallback(() => {
     setConfirmChanges(!confirmChanges);
-    setCreateError(false);
+    setApplyChangesErrorMessage(null);
   }, [confirmChanges, setConfirmChanges]);
 
   useEffect(() => {
-    setCreateError(createEntitiesError);
-  }, [createEntitiesError, setCreateError]);
+    setApplyChangesErrorMessage(formatError(applyChangesError));
+  }, [applyChangesError]);
```

Comment:
Don't you need to add `setApplyChangesErrorMessage` to the useEffect deps?

Reply:
![image](https://github.com/amplication/amplication/assets/1912933/a989862b-e104-413c-b74d-dabdddbfc9de)